iidm/iidm-api/src/main/javadoc/com/powsybl/iidm/network/doc-files/currentLimits.svg

```diff
@@ -11,93 +11,93 @@ file, You can obtain one at http://mozilla.org/MPL/2.0/.
 <!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
You should revert this change

Reply:
Ouch, I already reverted it, but got mixed up apparently!